_includes/indextable.html

```diff
@@ -25,6 +21,20 @@ <h2 id="{{status|slugify}}">{{status}}</h2>
           {% endif %}
         </thead>
         {% for page in rskips %}
+        {% if status == "Purpose and Guidelines" %}
+        <tr>
+          <td class="ipnum"><a href="{{page.url|relative_url}}">{{page.rskip|xml_escape}}</a></td>
+          {% if status == "Last Call" and page.last-call-deadline != undefined %}
```

Comment:
This "if status == Last..." will never be true, bc is inside another "if status == Purpose...". Am i missing something?

Reply:
Also, I cannot see any diff between both blocks inside the if
if
 block1
else
 block2
 
 block1 and block2 have the same code.

Reply:
Right, we're not using the status “last call” and there is some redundant code. I've cleaned it up and removed the unnecessary code. I've updated the Readme to include instructions on how to build the site locally to test. Thanks for the suggestion! I've already modified RSKIP0 and RSKIP0_TEMPLATE  to include the updated instructions on how to add the new IP using Jekyll. Basically, the main difference would be they would need to add the Jekyll front matter by adding three hyphens before and after the title, created date, author, etc.  Really the process is quite similar but if it's still not clear please let me know.